shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/spi/singleton/SingletonServiceLoaderTest.java

```diff
@@ -26,9 +27,10 @@
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertThat;
 import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
 
 public final class SingletonServiceLoaderTest {
-    
+
```

Comment:
Please do not change the original intent, please update your IDEA code format settings

Reply:
The idea configuration imported through the official website seems to be wrong.